plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/triggers/get_incidents/trigger.py

```diff
@@ -20,10 +20,9 @@ def __init__(self):
     def run(self, params={}):
         time_field = "creation_time"
 
-        # We will be asking for incidents between one hour ago and now.
-        one_hour_in_ms = 60 * 60 * 1000
         end_time = Util.now_ms()
-        start_time = end_time - one_hour_in_ms
+        # Request all events between 1ms ago and now on first iteration.
+        start_time = end_time - 1
```

Comment:
Why the change in functionality? @jmcadams-r7

Reply:
@jmcadams-r7 said that we wanted to start getting incidents/alerts from when the trigger triggered and onwards, not to start from an hour before the trigger triggered

Reply:
The way it was coded before would have returned all alerts from an hour before the trigger was started on the first loop. That could potentially be a ton of data, on top of it not being expected behavior.